### ISSUE TITLE
Changing Wing Segment Type

### ISSUE THREAD

**Issue by mclarke2** [suavecode/SUAVE#316]
Currently, wing segments have a different type than wings and control surfaces. There must be consistency in this for future users of SUAVE (within ADL and the general community)

**Comment by timdmacdo** [comment]
Can you explain this in more detail? What exactly needs to be consistent and why?

**Comment by mclarke2** [comment]
We can do "for wing in vehicle.wings" where "wing" in a class not a pointer. i.e. I can write
```
for wing in wings:
    wing.tag  = 'horizontal tail'
    wing.area = 0.4
```
however, if I want to parse through a segment within a wing, the argument is a pointer I write
for wing in wings: 
    area = wing.span * wing.chord  
    for seg in wing.segments:
          seg_area = wing.span* wing[**'seg'**].percent.chord. 

it should be 
seg_area = wing.span* seg.percent.chord.

**Comment by rafaellehmkuhl** [comment]
Has this already been adressed?
I think I can help.

**Comment by mclarke2** [comment]
Addressed in #349
